src/jobs/lint.yml

```
@@ -46,7 +46,9 @@ steps:
         - run:
             name: Run Linting
             command: |-
-              npm run lint
+              npm run lint -- --format junit -o reports/junit/js-lint-results.xml
```

```suggestion
              npm run lint -- --format junit -o reports/junit/js-lint-results.xml
```
`lint -- --format` are you missing a flag?